docs/commands/rhoas_service-registry_artifact.adoc

```diff
@@ -0,0 +1,156 @@
+ifdef::env-github,env-browser[:context: cmd]
+[id='ref-rhoas-service-registry-artifact_{context}']
+= rhoas service-registry artifact
+
+[role="_abstract"]
+[alpha] Manage Service Registry artifacts
+
+[discrete]
+== Synopsis
+
+Manage Service Registry schema and API artifacts in the currently selected Service Registry instance.
+
+Commands are executed on the currently selected Service Registry instance, which can be overridden using the --instance-id flag.
+
+Service Registry enables developers to manage and share the structure of their data.
+For example, client applications can dynamically push or pull the latest schema or API updates to or from the registry without needing to redeploy.
+Service Registry also enables developers to create rules that govern how registry content can evolve over time.
+For example, this includes rules for content validation and version compatibility.
+
+Artifact commands enable client applications to manage the schema and API artifacts in the registry instance.
+This set of commands provide create, read, update, and delete operations for artifacts, rules, versions, and metadata.
+
+This command is available as part of the developer preview
```

Comment:
Suggest the following (occurs in multiple files):
This command is available as part of the Development Preview release.